framework_tvb/tvb/adapters/analyzers/bct_adapters.py

```diff
@@ -183,11 +204,16 @@ class ModularityOCSM(BaseBCT):
     _ui_description = bct_description("modularity_dir.m")
     _matlab_code = "[Ci,Q] = modularity_dir(CW);"
 
-    def launch(self, connectivity, **kwargs):
+    def get_form_class(self):
+        return ModularityOCSMForm
+
+    def launch(self, view_model):
         # Prepare parameters
-        kwargs['CW'] = connectivity.weights
+        connectivity = self.get_connectivity(view_model)
+        data = dict([('CW', connectivity.weights)])
```

Comment:
Shorter to do just: data = {'CW': connectivity.weights} since there is only one entry for every bct adapter

Reply:
made changes so it would be shorter